main.py

```diff
@@ -1,5 +1,5 @@
 import matplotlib
-# matplotlib.use('Agg') # needed for hpc batch sims
+matplotlib.use('Agg') # needed for hpc batch sims
```

Comment:
since `Agg` has no UI I think it should be kept as-is

Reply:
fixed